docsets/Xojo/docset.json

```diff
@@ -0,0 +1,21 @@
+{
+    "name": "Xojo 2023r4",
```

Comment:
The name of the docset should not include the version

Reply:
I'm happy to change this to just Xojo, but alas it's an interesting beast in that the documentation is tied to the actual release version of the product. So although I can change to just Xojo, the Docset only applies to the 2023 release 4 version of the product. See my next comment below.

Reply:
Docset versioning is handled by the `version` and not the docset name/other places.